nrt/data/HVDC/HVDC.dyd

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+    Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+    simulation tool for power systems.
+-->
+<dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:modelicaModel id="InfBus1">
+    <dyn:unitDynamicModel id="Bus" name="Dynawo.Electrical.Buses.InfiniteBus" parFile="./HVDC.par" parId="1"/>
+  </dyn:modelicaModel>
+  <dyn:modelicaModel id="InfBus2">
+    <dyn:unitDynamicModel id="Bus" name="Dynawo.Electrical.Buses.InfiniteBus" parFile="./HVDC.par" parId="2"/>
+  </dyn:modelicaModel>
```

Comment:
Your test case makes me realized that we don't have any preassembled model for InfiniteBus. I just open an issue for this (#777 ). If it is done and merged before your PR, you could use it here. Otherwise that's fine too.

Reply:
done